PWGLF/DataModel/LFStrangenessTables.h

```diff
@@ -565,45 +565,103 @@ using V0Linked = V0sLinked::iterator;
+                                                                      ((1 << enums::bitdEdxPosProton) | (1 << enums::bitdEdxNegPion)); });
+DECLARE_SOA_DYNAMIC_COLUMN(IsdEdxAntiLambda, isdEdxAntiLambda, //!
+                           [](uint32_t selectionMap) -> bool { return (selectionMap &
+                                                                       ((1 << enums::bitdEdxPosPion) | (1 << enums::bitdEdxNegProton))) ==
+                                                                      ((1 << enums::bitdEdxPosPion) | (1 << enums::bitdEdxNegProton)); });
+DECLARE_SOA_DYNAMIC_COLUMN(IsdEdxHypertriton, isdEdxHypertriton, //!
+                           [](uint32_t selectionMap) -> bool { return (selectionMap &
+                                                                       ((1 << enums::bitdEdxPosHelium3) | (1 << enums::bitdEdxNegPion))) ==
+                                                                      ((1 << enums::bitdEdxPosHelium3) | (1 << enums::bitdEdxNegPion)); });
+DECLARE_SOA_DYNAMIC_COLUMN(IsdEdxAntiHypertriton, isdEdxAntiHypertriton, //!
+                           [](uint32_t selectionMap) -> bool { return (selectionMap &
+                                                                       ((1 << enums::bitdEdxPosPion) | (1 << enums::bitdEdxNegHelium3))) ==
+                                                                      ((1 << enums::bitdEdxPosPion) | (1 << enums::bitdEdxNegHelium3)); });
+
+DECLARE_SOA_DYNAMIC_COLUMN(IsTrueGamma, isTrueGamma, //!
+                           [](uint32_t selectionMap) -> bool { return (selectionMap & (1 << enums::bitTrueGamma)) == (1 << enums::bitTrueGamma); });
+DECLARE_SOA_DYNAMIC_COLUMN(IsTrueK0Short, isTrueK0Short, //!
+                           [](uint32_t selectionMap) -> bool { return (selectionMap & (1 << enums::bitTrueK0Short)) == (1 << enums::bitTrueK0Short); });
+DECLARE_SOA_DYNAMIC_COLUMN(IsTrueLambda, isTrueLambda, //!
+                           [](uint32_t selectionMap) -> bool { return (selectionMap & (1 << enums::bitTrueLambda)) == (1 << enums::bitTrueLambda); });
+DECLARE_SOA_DYNAMIC_COLUMN(IsTrueAntiLambda, isTrueAntiLambda, //!
+                           [](uint32_t selectionMap) -> bool { return (selectionMap & (1 << enums::bitTrueAntiLambda)) == (1 << enums::bitTrueAntiLambda); });
+DECLARE_SOA_DYNAMIC_COLUMN(IsTrueHypertriton, isTrueHypertriton, //!
+                           [](uint32_t selectionMap) -> bool { return (selectionMap & (1 << enums::bitTrueHypertriton)) == (1 << enums::bitTrueHypertriton); });
+DECLARE_SOA_DYNAMIC_COLUMN(IsTrueAntiHypertriton, isTrueAntiHypertriton, //!
+                           [](uint32_t selectionMap) -> bool { return (selectionMap & (1 << enums::bitTrueAntiHypertriton)) == (1 << enums::bitTrueAntiHypertriton); });
+
+DECLARE_SOA_COLUMN(Tag, tag, uint32_t); // pack according to enums
```

Comment:
There is DECLARE_SOA_BITMAP_COLUMN, that hold an unsigned int (of 8, 16, 32 or 64 bit) and has `getter_bit(pos)` defined that returns bool. If you make the enum convertible to int, you can just do `if (tag_bit(bitTrueLambda))` for example.

Reply:
Yes, that's a good point, indeed, I'll move to that shortly :-D thanks!

Reply:
Hi @aalkin one curiosity though: this will just offer an interface for checking the bits, right? There is no inherent speed advantage in this, right?

Reply:
Dynamic columns are not very fast in general, as they contain lambdas, however in this case those are probably optimized out by the compiler so that there is no performance overhead. But you can skip adding a bunch of dynamic columns in the data model and the syntax becomes a bit simpler.

Reply:
Now implemented! Thanks :-D